apps/builder/app/builder/features/style-panel/shared/css-value-input/css-value-input.tsx

```diff
@@ -932,6 +932,7 @@ export const CssValueInput = ({
                             {toValue(item.fallback)}
                           </Text>
                         )}
+                        {/* @todo */}
```

Comment:
better write `@todo colors` or something to be able to find it

Reply:
Left it so you notice it and remind me to fix it 😅